docs/en/latest/deployments/ack.md

```diff
@@ -38,6 +38,7 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 ```shell
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
 helm dependency update ./charts/apisix
```

Comment:
Since we use helm repo, It is recommended to modify like this.
```suggestion
helm search repo apisix
helm repo update
helm install apisix apisix/apisix \
  --set ...
```